docs/qcfractal/source/managers.rst

```diff
@@ -0,0 +1,216 @@
+      --config-file CONFIG_FILE
+
+    Common Adapter Settings:
+      --adapter ADAPTER     The backend adapter to use, currently only {'dask',
+                            'parsl', 'pool'} are valid.
+      --ntasks NTASKS       The number of simultaneous tasks for the executor to
+                            run, resources will be divided evenly.
+      --ncores NCORES       The number of process for the executor
+      --memory MEMORY       The total amount of memory on the system in GB
+      --scratch-directory SCRATCH_DIRECTORY
+                            Scratch directory location
+      -v, --verbose         Increase verbosity of the logger.
+
+    FractalServer connection settings:
+      --fractal-uri FRACTAL_URI
+                            FractalServer location to pull from
+      -u USERNAME, --username USERNAME
+                            FractalServer username
+      -p PASSWORD, --password PASSWORD
+                            FractalServer password
+      --verify VERIFY       Do verify the SSL certificate, turn off for servers
+                            with custom SSL certificiates.
+
+    QueueManager settings:
+      --max-tasks MAX_TASKS
+                            Maximum number of tasks to hold at any given time.
+      --manager-name MANAGER_NAME
+                            The name of the manager to start
+      --queue-tag QUEUE_TAG
+                            The queue tag to pull from
+      --log-file-prefix LOG_FILE_PREFIX
+                            The path prefix of the logfile to write to.
+      --update-frequency UPDATE_FREQUENCY
+                            The frequency in seconds to check for complete tasks.
+
+    Optional Settings:
+      --test                Boot and run a short test suite to validate setup
+      --ntests NTESTS       How many tests per found program to run, does nothing
+                            without --test set
+
+
+Terminology
+-----------
+
+There are a number of terms which can overlap in due to the layers of abstraction and the type of software and hardware
+the Queue Manager interacts with. To help with that, the pages in this section will use the following terminology.
+Several pieces of software we interface with may have their own terms or the same term with different meaning, but
+because one goal of the Manager is to abstract those concepts away as best it can, we choose the following set. If
+you find something inappropriately labeled, unclear, or overloaded in any way, please raise an issue
+`on GitHub <https://github.com/MolSSI/QCFractal/issues/new/choose>`_ and help us make it better!
+
+An important note: Not all the concepts/mechanics of the :term:`Manager` and :term:`Adapter` are covered here by design!
+There are several abstraction layers and mechanics which the user should never have to interact with or even be aware
+of. However, if you feel something is missing, let us know!
+
+.. glossary::
```

Comment:
Should these go in the primary glossary or can we cross reference a glossary in any file?